plugin/pxr/maya/lib/usdMaya/CMakeLists.txt

```diff
@@ -26,7 +25,6 @@ pxr_shared_library(${PXR_PACKAGE}
         mayaUsd
 
     INCLUDE_DIRS
-        ${Boost_INCLUDE_DIRS}
```

Comment:
these variables are empty?? The mystery part is how usdMaya links against Boost?